### PR TITLE
Handle dashboard state when all assets are hidden

### PR DESCRIPTION
**Pull request by blurpesec**
### Description
Introduced a new state of the dashboard for when all assets are hidden.

image

### CODE REVIEW

File: src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx [hunk 1]
```diff
@@ -281,7 +281,12 @@ export default function WalletBreakdownView({
       .dividedBy(new BigNumber(totalFiatValue))
       .multipliedBy(100)
       .toFixed(2);
-
+  console.debug(
+    'chartBalances',
+    chartBalances.length,
+    'breakdownBalances: ',
+    breakdownBalances.length
+  );
```
Comment (vilsbole) [inline src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx:289]: Do we need to keep this ?
If so prefer `logger` from `@utils`

File: src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx [hunk 2]
```diff
@@ -281,7 +281,12 @@ export default function WalletBreakdownView({
       .dividedBy(new BigNumber(totalFiatValue))
       .multipliedBy(100)
       .toFixed(2);
-
+  console.debug(
```
Comment (FrederikBolding) [inline src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx:284]: Remove?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Looks good except for one small thing

File: src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx (hunk 1 above)
Comment (FrederikBolding) [reply src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx:289]: Didn't see this 😆

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**